### PR TITLE
Added create and edit CRUD to blogs in the admin panel

### PR DESCRIPTION
**Pull request by AlexGit123**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
 });
 
 
-Route::resource('/blog', BlogController::class);
```
Comment (koenhamstra) [inline routes/web.php:25]: Why is there blog and blogs for the same controller?

**Review by koenhamstra — APPROVED** [review]
Looks good

File: routes/web.php (hunk 1 above)
Comment (awqfrxx) [reply routes/web.php:25]: I have told Koen about this, don't worry Alex

File: resources/views/blogs/create.blade.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
                                 <select class="select-education @error('course') is-danger @enderror" type="text"
                                     id="course" name="course">
                                     <option value="{{ old('none') }}">None</option>
-                                    @foreach ($course as $course)
+                                    @foreach ($courses as $course)
```
Comment (awqfrxx) [inline resources/views/blogs/create.blade.php]: You give $course through the blogcontroller but here u use $courses, so it doesn't work and give an error when u click on the Blog in the navbar

File: resources/views/blogs/edit.blade.php [hunk 3]
```diff
@@ -0,0 +1,234 @@
+@extends('common.master')
+
+@section('content')
+    <section class="hero  is-medium  is-bold">
+        <div class="header-img">
+            <img src="./image/sdg-header-en.png" />
```
Comment (koenhamstra) [inline resources/views/blogs/edit.blade.php]: SDG tool image wont show
remove the . before ./image/sdg-header-en.png

**Review by koenhamstra — CHANGES_REQUESTED** [review]
rest looks good

**Pull request merged**